pymatgen/io/zeopp.py

```diff
@@ -117,7 +117,8 @@ def from_str(cls, string: str) -> Self:
         lengths.insert(0, a)
         alpha = angles.pop(-1)
         angles.insert(0, alpha)
-        lattice = Lattice.from_parameters(*lengths, *angles)
```

Comment:
Not sure if there is a better way out. It seems we have multiple `mypy` complaints related to unpacking args.